src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php

```diff
@@ -377,4 +377,20 @@ protected function isCsrfTokenValid(string $id, string $token): bool
 
         return $this->container->get('security.csrf.token_manager')->isTokenValid(new CsrfToken($id, $token));
     }
+
+    /**
+     * Gets a container configuration parameter by its name.
+     *
+     * @return mixed
+     *
+     * @final since version 4.1
```

Comment:
just `@final` (since always - it didn't exist before)

Reply:
fixed